client/state/ui/guided-tours/contexts.js

```diff
@@ -26,6 +27,16 @@ export const isNewUser = state => {
 	return ( Date.now() - creation ) <= WEEK_IN_MILLISECONDS;
 };
 
+export const userIsOlderThan = age => state => {
```

Comment:
Can we deduplicate some of the code that `userIsOlderThan` and `isNewUser` share?